### PR TITLE
Styling of login page

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/common/AppFooter/AppFooter.jsx [hunk 1]
```diff
@@ -11,49 +11,67 @@ import Breakpoint from '../Breakpoint/';
 const renderDesktopFooter = (props) => {
   if (props.routeName === 'Login') {
     return (
-      <div className={'footer-container'}>
+      <div className={'footer-container-login'}>
+        <p class={'footer-account-link'}>Don't have an account?
+        <a href={'#'}>Create Account</a>
```
Comment (ayushmalik2110) [inline src/components/common/AppFooter/AppFooter.jsx:16]: Please do not use anchor tag. Use <Link>instead.

File: src/components/routes/Member/Login/MemberLoginForm/_MemberLoginForm.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  flex-direction: column;
+  margin-top: 100px;
+  margin-bottom: 100px;
+}
+fieldset{
+  border: none;
```
Comment (ayushmalik2110) [inline src/components/routes/Member/Login/MemberLoginForm/_MemberLoginForm.scss:11]: Use property 'unset'

File: src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMobileMenuProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Icon from '../../../Typography/Icon';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+
+const AppFooterMobileMenuProfile = () =>
+  <div className={"app-footer__mobile__menu__item app-footer__mobile__menu__profile"}>
+    {/*Todo uncomment the below code with member profile image is available*/}
```
Comment (mohitsaladiya) [inline src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMobileMenuProfile.jsx:8]: Where is the below code to uncomment?

File: src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMenuMobile.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  from '../../../appHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderDesktopMenuProfile';
+import AppFooterMobileMenuSetting from './AppFooterMobileMenuSetting';
+import AppFooterMobileMenuHome from './AppFooterMobileMenuHome';
+
+/**
+ * This component is footer menu for mobile
+ * @returns {*}
+ * @constructor
+ */
+const AppFooterMenuMobile = () =>
```
Comment (mohitsaladiya) [inline src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMenuMobile.jsx:18]: Make this things config driven

File: src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMobileMenuProfile.jsx (hunk 3 above)
Comment (ayushmalik2110) [reply src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMobileMenuProfile.jsx]: Corrected.

File: src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMenuMobile.jsx (hunk 4 above)
Comment (ayushmalik2110) [reply src/components/common/AppFooter/AppFooterMobile/AppFooterMenuMobile/AppFooterMenuMobile.jsx:18]: Will do in next pr.

**Pull request merged**